src/App.js

```diff
@@ -4,15 +4,57 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>
-          Eric F. Olsen
-        </h1>
-        <h3>
-          Software Engineer
-        </h3>
-      </header>
+      <Header />
+      {/* <Story />
+      <Resume />
+      <Projects />
+      <Testimonials /> */}
+    </div>
+  );
+}
+
+function Header() {
```

Comment:
I would move each of these to their own files.  If you create a Header.js  file and `export default Header` at the bottom, you can import it at the top of this file like this `import Header from './Header';` and then use it like you're using it here.

Reply:
Thanks for the suggestion!